address_book.rb

```diff
@@ -1,13 +1,26 @@
 class AddressBook
   def initialize(csv_path)
-
+    require 'csv'
```

Comment:
```require```'in dosyanin basinda olmasi daha uygun.

```@@book``` yaptigin zaman bu bi class degiskeni oluyor. Yani diyelim ki projede 2 tane farkli AddressBook var, ```@@book``` kullandigin zaman ikiside ayni degeri kullanacak.

```ruby
a1 = AddressBook.new('x.csv')
a2 =AddresBook.new('y.csv')
```

Yukaridaki durumda a1'in `x.csv`'yi kullanmasi gerekiyor ama @@book dedigin icin ikisi de `y.csv`yi kullaniyor.